lib/engine/step/special_token.rb

```diff
@@ -60,11 +60,12 @@ def process_place_token(action)
             true
           end
 
+        connected = special_ability.type == :token ? special_ability.reachable : false
```

Comment:
In graph, reachable and connected are two different concepts. I believe you want connected here (there is connected track between the destination city and another tokened city), so the ability should be renamed to connected.

Reply:
Okay, I have them backwards. Then the kwarg should be called reachable, not connected. If that's easy to fix, can you fix it?

Reply:
Sure.

Reply:
Although I'm not sure if that's the right choice. In `process_place_token()` we operate on hex and use `reachable_hexes`. However, `place_token()` (the method with the `connected` kwarg) operates on city and uses `connected_nodes`. So I think that the name of the kwarg makes sense.

Reply:
Yeah, I'm good with that.